cmd/snap/cmd_services.go

```diff
@@ -136,21 +167,9 @@ func (s *svcStatus) Execute(args []string) error {
 	defer w.Flush()
 
 	fmt.Fprintln(w, i18n.G("Service\tStartup\tCurrent\tNotes"))
-
 	for _, svc := range services {
-		startup := i18n.G("disabled")
-		if svc.Enabled {
-			startup = i18n.G("enabled")
-		}
-		current := i18n.G("inactive")
-		if svc.DaemonScope == snap.UserDaemon {
-			current = "-"
-		} else if svc.Active {
-			current = i18n.G("active")
-		}
-		fmt.Fprintf(w, "%s.%s\t%s\t%s\t%s\n", svc.Snap, svc.Name, startup, current, clientutil.ClientAppInfoNotes(svc))
+		fmt.Fprintf(w, fmtServiceStatus(svc, u.Uid == "0" || s.Global))
```

Comment:
it's a bit annoying that we need to encode the definition of isGlobal both here and in daemon